css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+#main {
+background-color: #EEEEEE;
+padding: 2%;
+}
```

Comment:
Kindly suggest that you use 2 spaces of indentation on your code.
This will help in the visual aspect of your code ☺️